src/plots/smith/layout_attributes.js

```diff
@@ -0,0 +1,206 @@
+}, 'plot', 'from-root');
+
+var radialAxisAttrs = {
+    visible: extendFlat({}, axesAttrs.visible, {dflt: true}),
+
+    title: {
+        // radial title is not gui-editable at the moment,
+        // so it needs dflt: '', similar to carpet axes.
+        text: extendFlat({}, axesAttrs.title.text, {editType: 'plot', dflt: ''}),
+        font: extendFlat({}, axesAttrs.title.font, {editType: 'plot'}),
+
+        // TODO
+        // - might need a 'titleside' and even 'titledirection' down the road
+        // - what about standoff ??
+
+        editType: 'plot'
+    },
+
+    hoverformat: axesAttrs.hoverformat,
+
+    uirevision: {
+        valType: 'any',
+        editType: 'none',
+        description: [
+            'Controls persistence of user-driven changes in axis `range`,',
+            '`autorange`, `angle`, and `title` if in `editable: true` configuration.',
+            'Defaults to `smith<N>.uirevision`.'
+        ].join(' ')
+    },
+
+    editType: 'calc',
+};
+
+extendFlat(
+    radialAxisAttrs,
+
+    // N.B. realaxis grid lines are circular,
+    // but realaxis lines are straight from circle center to outer bound
+    axisLineGridAttr,
+    axisTickAttrs
+);
+
+var angularAxisAttrs = {
+    visible: extendFlat({}, axesAttrs.visible, {dflt: true}),
+
+    direction: {
+        valType: 'enumerated',
+        values: ['counterclockwise', 'clockwise'],
+        dflt: 'counterclockwise',
+        editType: 'calc',
+        description: [
+            'Sets the direction corresponding to positive angles.'
+        ].join(' ')
+    },
+
+    rotation: {
+        valType: 'angle',
+        editType: 'calc',
+        description: [
+            'Sets that start position (in degrees) of the angular axis',
+            'By default, smith subplots with `direction` set to *counterclockwise*',
+            'get a `rotation` of *0*',
+            'which corresponds to due East (like what mathematicians prefer).',
+            'In turn, smith with `direction` set to *clockwise* get a rotation of *90*',
+            'which corresponds to due North (like on a compass),'
+        ].join(' ')
+    },
+
+    hoverformat: axesAttrs.hoverformat,
+
+    uirevision: {
+        valType: 'any',
+        editType: 'none',
+        description: [
+            'Controls persistence of user-driven changes in axis `rotation`.',
+            'Defaults to `smith<N>.uirevision`.'
+        ].join(' ')
+    },
+
+    editType: 'calc'
+};
+
+extendFlat(
+    angularAxisAttrs,
+
+    // N.B. angular grid lines are straight lines from circle center to outer bound
+    // the angular line is circular bounding the smith plot area.
+    axisLineGridAttr,
+
+    // N.B. ticksuffix defaults to '°' for angular axes with `thetaunit: 'degrees'`
+    axisTickAttrs
+);
+
+module.exports = {
+    // TODO for x/y/zoom system for paper-based zooming:
+    // x: {},
+    // y: {},
+    // zoom: {},
+
+    domain: domainAttrs({name: 'smith', editType: 'plot'}),
+
+    sector: {
```

Comment:
`sector` is supported?

Reply:
8ace040ea4abbb87d6e2750b464e6e8e28334a2a